components/Chat/Frame/FramePreview.tsx

```diff
@@ -152,7 +152,9 @@ export default function FramePreview({
         const link = button.target;
         if (
           !link ||
-          !link.startsWith("http") ||
+          !(
+            link.startsWith("http") || link.startsWith(`${config.scheme}://`)
```

Comment:
I've been burned in the past with manually string matching URLs. Could we add the react-native-url-polyfill and perform parsing with it instead?